index.js

```diff
@@ -4,7 +4,7 @@ import MochaAdapter from './lib/adapters/MochaAdapter.js'
 import TapReporter from './lib/reporters/TapReporter.js'
 import ConsoleReporter from './lib/reporters/ConsoleReporter.js'
 import {Assertion, Test, Suite} from './lib/Data.js'
-import {autoRegister} from './lib/helpers.js'
```

Comment:
I don't understand this change: What was the motivation? This looks worse to me, compared to what it was before.